pkg/server/skyerr/skyerr.go

```diff
@@ -119,6 +119,9 @@ const (
 	// PluginTimeout occurs when an operation carried by a plugin is timed out
 	PluginTimeout
 
+	// PluginInitializing occurs when any of the plugins are initializing
```

Comment:
“Error codes for expected error condition should be placed above this line.”

i.e. do not insert errors in the middle of a list